### PR TITLE
Migrate to api token

### PR DESCRIPTION
**Pull request by viktor-orca**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [x] Yes
       - Require parameters changed
       - API Key replaced by API Token
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by JasBeilin** [comment]
Hi @viktor-orca, how are you? This PR is in draft and will not be reviewed, If this is a mistake and the PR is ready for review, please let us know.

**Comment by ShacharKidor** [comment]
Hi @viktor-orca, 
Thank you for your contribution!

As @JasBeilin asked before, we can see that the PR is in Draft - are you still working on it? or we can start the review? 

In addition, please fill in the following registration form:
> Thank you for your contribution. Your generosity and caring are unrivaled! Make sure to register your contribution by filling the [Contribution Registration](https://forms.gle/XDfxU4E61ZwEESSMA) form, so our content wizard @JasBeilin will know he can start review the proposed changes.

Thanks again

**Comment by JasBeilin** [comment]
Hi @viktor-orca 
As this is been in draft mode for more than 2 weeks now without any response, I am closing this PR. Feel free to re-open once you want to move forward.

**Comment by viktor-orca** [comment]
Hi @JasBeilin Thank you for your attention.
Sorry for the no responses. I was on vacation and we waited for update UI on our side.
I'm going to reopen this pr and put it to review

**Comment by ShacharKidor** [comment]
Hi @viktor-orca - No problem :) feel free to reopen.

**Comment by viktor-orca** [comment]
Hi @ShacharKidor 
Thanks for information
I just created new pr https://github.com/demisto/content/pull/21481
I will close new pr and leave only this.

File: Packs/Orca/Integrations/Orca/Orca.yml [hunk 1]
```diff
@@ -3,8 +3,8 @@ commonfields:
   id: Orca
   version: -1
 configuration:
-- display: API Key
-  name: apikey
+- display: API Token
+  name: api_token
   required: true
   type: 4
```
Comment (yuri-orca) [inline Packs/Orca/Integrations/Orca/Orca.yml:9]: per discussion with @edik24 , change to `type: 9`
```suggestion
  type: 9
```

File: Packs/Orca/Integrations/Orca/Orca.yml [hunk 2]
```diff
@@ -3,10 +3,10 @@ commonfields:
   id: Orca
   version: -1
 configuration:
-- display: API Key
-  name: apikey
+- display: API Token
```
Comment (moishce) [inline Packs/Orca/Integrations/Orca/Orca.yml:6]:
```suggestion
- displaypassword: API Token
  hiddenusername: true
```
and to get the data in the code `demisto.params().get('api_token').get('password')`

File: Packs/Orca/Integrations/Orca/Orca.py [hunk 3]
```diff
@@ -246,12 +246,11 @@ def main() -> None:
     try:
         command = demisto.command()
         demisto.debug(f'Orca Command being called is {command}')
-        api_key = demisto.params().get('apikey')
+        api_token = demisto.params().get('api_token')
```
Comment (moishce) [inline Packs/Orca/Integrations/Orca/Orca.py:249]:
```suggestion
        api_token = demisto.params().get('api_token').get('password')
```

**Review by moishce — COMMENTED** [review]
Thank you for your contribution!
Some small changes.

File: Packs/Orca/ReleaseNotes/2_0_0.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+##### Orca
+- Updated the Docker image to: *demisto/python3:3.10.7.33922*.
+- Remove support of the API key
+- Add support API token
+- Change used api for fetch alerts
```
[the remark this answers is not in the document]
Comment (viktor-orca) [reply Packs/Orca/ReleaseNotes/2_0_0.md]: done

File: Packs/Orca/ReleaseNotes/2_0_0.md [hunk 5]
```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+##### Orca
+- Updated the Docker image to: *demisto/python3:3.10.7.33922*.
+- Remove support of the API key
+- Add support API token
```
[the remark this answers is not in the document]
Comment (viktor-orca) [reply Packs/Orca/ReleaseNotes/2_0_0.md]: done

File: Packs/Orca/Integrations/Orca/Orca.yml (hunk 2 above)
Comment (viktor-orca) [reply Packs/Orca/Integrations/Orca/Orca.yml]: done

File: Packs/Orca/Integrations/Orca/Orca.py (hunk 3 above)
Comment (viktor-orca) [reply Packs/Orca/Integrations/Orca/Orca.py]: done

**Comment by moishce** [comment]
When there are breaking changes, you also need a JSON file in the release notes,
So please rerun the update-release-notes with the -bc flag
you can see an explanation [here](https://xsoar.pan.dev/docs/documentation/release-notes#breaking-changes-version)

**Review by moishce — APPROVED** [review]

**Pull request merged**